moonrunners/chains.json

```diff
@@ -0,0 +1,15 @@
+{
+  "$schema": "../chains.schema.json",
+  "name": "MoonRunners",
+  "chains": [
+    {
+      "name": "terra",
+      "address": "terravaloper1epeqf5t6kj7wdr3zwqu38z4wayay3e6vp9s5m2",
+      "restake": {
+        "address": "terra10rrp2x53kau5e6cupzuh7dfhngzmujkd700she",
+        "run_time": "Every 24 hours",
```

Comment:
You should specify a run time for every 24 hours, in UTC. E.g. `"run_time": "09:00"`. You can also use an array for multiple times